masonry_core/src/passes/event.rs

```diff
@@ -21,7 +21,9 @@ fn get_pointer_target(
 ) -> Option<WidgetId> {
     // See the [pointer capture documentation](../doc/06_masonry_concepts.md#pointer-capture).
     if let Some(capture_target) = root.global_state.pointer_capture_target {
-        return Some(capture_target);
+        if root.is_still_interactive(capture_target) {
+            return Some(capture_target);
+        }
```

Comment:
Discussion: Should we remove the pointer capture target in the else here?

Determining that needn't block this PR